### ISSUE TITLE
Using Inspryrenet Rembg, is there a node I can add to tweak the granularity of the crops?

### ISSUE THREAD

**Issue by vitalbuttering** [john-mnz/ComfyUI-Inspyrenet-Rembg#16]
I am still seeing some miscrops using the Inspyrenet Rembg. Is there a node I can add to tweak settings and improve the granularity of the crop details?

**Comment by john-mnz** [comment]
have you tried playing with the threshold in the advanced node?

**Comment by vitalbuttering** [comment]
> have you tried playing with the threshold in the advanced node?

yep i did. still not as clean though, even by tweaking the thresholds. there are still some bleeding after the removal.

**Comment by CognitiveDiffusion** [comment]
Came here to say the same.
I wanted to tweak some images manually, but the crop of the advanced node (even at 0.01) is very aliased compared to the nice and soft edges of the Auto-node.

Thanks a lot for your work - it's much appreciated.
